test/acceptance/features/environment.py

```diff
@@ -49,9 +55,35 @@ def before_all(_context):
         _context.sbo_namespace = output
     else:
         assert False, f"TEST_ACCEPTANCE_START_SBO={start_sbo} is currently unsupported."
+    _context.no_scenarios_failed = True
 
 
 def before_scenario(_context, _scenario):
     _context.bindings = dict()
     output, code = cmd.run(f'{ctx.cli} get ns default -o jsonpath="{{.metadata.name}}"')
     assert code == 0, f"Checking connection to OS cluster by getting the 'default' project failed: {output}"
+
+
+def after_scenario(_context, scenario):
+    if "namespace" in _context:
+        namespace = _context.namespace.name
+    elif os.getenv("TEST_NAMESPACE"):
+        namespace = os.getenv("TEST_NAMESPACE")
+    else:
+        print(f"No namespace set in context nor TEST_NAMESPACE env variable is set, skipping deletion")
+        return
+    
+    if delete_namespace == "alwaysdelete":
+        output, code = cmd.run(f"{ctx.cli} delete namespace {namespace} --ignore-not-found.")
```

Comment:
Typo (the `.`) in the command
```suggestion
        output, code = cmd.run(f"{ctx.cli} delete namespace {namespace} --ignore-not-found")
```